### PR TITLE
Login page resolving

### PR DESCRIPTION
**Pull request by dharambir**
Before login :-
screenshot from 2018-10-09 16-55-22
After Login :-
screenshot from 2018-10-09 16-55-28

### CODE REVIEW

**Comment by nisusam** [comment]
👍 For screenshots

**Comment by dharambir** [comment]
Thank you @nisusam 
I have a little knowledge about git.
Can you make me understand the thing.
like branches, creatting pull-request, push to git

**Comment by nisusam** [comment]
@dharambir Can you please remove the `sign-in` & `sign-up` links from `navbar-header` in this same PR?

Since now it is part of the Login Page

**Comment by dharambir** [comment]
Oh! yes

**Comment by nisusam** [comment]
> Thank you @nisusam
> I have a little knowledge about git.
> Can you make me understand the thing.
> like branches, creatting pull-request, push to git

Yup. You can go through https://git-scm.com/book/en/v2

**Comment by dharambir** [comment]
git add -A
git commit -m "Login page design and setup done"
git push -u mvp

I am using this command to push. Is it OK with the stuff.

**Comment by dharambir** [comment]
> @dharambir Can you please remove the `sign-in` & `sign-up` links from `navbar-header` in this same PR?
> 
> Since now it is part of the Login Page

Actually I am not getting you. Because if we delete it. From where user will be toggle to login.

**Comment by nisusam** [comment]
> git add -A
> git commit -m "Login page design and setup done"
> git push -u mvp
> 
> I am using this command to push. Is it OK with the stuff.

Right now, it is OK...
but when there are multiple team members working on the same branch, you must use
```
git pull
```
as well before pushing the changes to remote.

**Comment by dharambir** [comment]
Ok! Thank You! for this @nisusam

**Comment by nisusam** [comment]
> > @dharambir Can you please remove the `sign-in` & `sign-up` links from `navbar-header` in this same PR?
> > Since now it is part of the Login Page
> 
> Actually I am not getting you. Because if we delete it. From where user will be toggle to login.

Here I am talking about the links in the top black colored navbar. We need to remove them... 
screenshot from 2018-10-09 16-55-22
After sign_out user will be directly redirect to the `root_path` which is `sign_in_path`.

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,6 @@ log/*.log
 tmp/
 vm_provisioning/ssh/id_rsa
 vm_provisioning/ssh/id_rsa.pub
+vim .gitignore
```
Comment (budhram) [inline .gitignore]: `vim .gitignore` is not required.

File: .gitignore [hunk 2]
```diff
@@ -9,3 +9,6 @@ log/*.log
...
+logs.txt
+config/initializers/devise.rb
```
Comment (budhram) [inline .gitignore]: :+1:  Awesome. This is what is required.

**Comment by budhram** [comment]
> what if we hide this using JavaScript on particularly on this page?

@dharambir Remove else part from here 
https://github.com/shikshaplus/pareeksha-manch/blob/master/app/views/layouts/_nav_links_for_auth.html.erb

That's it.

**Comment by kusumeshu** [comment]
Closing as it is merged now.

@dharambir Thanks for the contributions :+1:

**Comment by gyangrg** [comment]
The PR is merged!

**Pull request closed without merging**